src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -40,6 +40,13 @@ import { bignumber } from 'mathjs';
 const showPerps = !isMainnet || isStaging;
 const showZero = isMainnet || isStaging;
 
+interface PagesProps {
```

Comment:
```suggestion
type PageProps = {
```

Reply:
Just to clarify - you can use either `interface` or `type` but if you wanted it to be an `interface`, it should've had a prefix (`I`).